src/app.rs

```diff
@@ -67,9 +67,9 @@ pub struct App<
 }
 
 /// No-op application initialization function.
-pub fn no_init<BankT, CustomT, WasmT, StakingT, DistrT, IbcT, GovT, StargateT>(
+pub fn no_init<BankT, ApiT, CustomT, WasmT, StakingT, DistrT, IbcT, GovT, StargateT>(
```

Comment:
very nit: Can we add it either in front or back? Now it's in between the router's generics.

Reply:
Placed in front ;-)